app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
```

Comment:
- [ ] As per the given _ERD_, the _Comment_ **association** should have _Comment_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.

Reply:
Thank you for letting me know. I did it.